st2actions/tests/integration/test_localrunner.py

```diff
@@ -0,0 +1,106 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import st2tests.config as tests_config
+tests_config.parse_args()
+
+import uuid
+
+from unittest2 import TestCase
+from st2actions.container.service import RunnerContainerService
+from st2actions.runners import localrunner
+from st2common.constants import action as action_constants
+from st2tests.fixturesloader import FixturesLoader
+
+
+class TestLocalShellRunner(TestCase):
+
+    fixtures_loader = FixturesLoader()
```

Comment:
Yeah. That would work great. In this case I was not really loading into DB so would not have use DbTestCase anyway.